bindist/README.md

```diff
@@ -0,0 +1,50 @@
+# Clash's release process
+
+Clash actively deploys to two platforms:
+
+ * [Hackage](http://hackage.haskell.org/package/clash-prelude), a source
+   distribution platform for Haskell applications.
+ * [The Snap Store](https://snapcraft.io/clash), a binary distribution platform
+   for most Linux distributions.
+
+Development happens on the `master` branch. Released versions live in `1.0`,
+`1.2`, etc. Changes made to these branches automatically get published every 
+night. This is achieved through [GitLab Pipeline Schedules](https://gitlab.com/clash-lang/clash-compiler/pipeline_schedules).
+(You need admin access to see that page.) At the time of writing it gets 
+triggered on `master` and `1.2`. For Hackage, a release candidate gets pushed out 
+with the version numbers in the various cabal files. Candidates can be previewed 
+on [a special Hackage page](http://hackage.haskell.org/package/clash-prelude/candidates/).
+For the Snap Store, an _edge_ release gets made when triggered on `master`. When
+triggered on any other branch a _beta_ release gets made. 
+
+## Branches overview
+* `master`: development branch
+  * Only members of the `devops` team can merge to it
+  
+* `[0-9].[0-9]`: release branches
+  * Only members of the `devops` team can merge to it
+  * Warning: matches `1.0`, `1.2`, but NOT `1.22` (GitHub limitation).
+ 
+## Releasing a new version
+1. Change version numbers in:
+  * `clash-prelude/clash-prelude.cabal`
+  * `clash-lib/clash-lib.cabal`
+  * `clash-ghc/clash-ghc.cabal`
+  * `bindist/linux/snap/snap/snapcraft.yaml`
+  * `docs/conf.py`
+  * `README.md`
+  * `content/install/index.md` @ `clash-lang/clash-lang.org`
+2. Change the defaults of cabal flags in `clash-prelude/clash-prelude.cabal`
+   where necessary. At the time of writing this applies to `-fmultiple-hidden`
+   and `-flarge-tuples`.
+3. Update the CHANGELOG (see changelog/README.md).
+4. Create a branch of the called ~`1.2`.
+5. Ask someone with admin permissions on GitLab to create a new nightly schedule
```

Comment:
Point 5 also doesn't say anything on how to do this. Maybe it's obvious, maybe it isn't?

Reply:
I've elaborated a bit, but it should be obvious what to do at https://gitlab.com/clash-lang/clash-compiler/pipeline_schedules :)

Reply:
I get a 404 at that link, but I'll trust when you say it's obvious enough

Reply:
Yeah, if you're not logged in with the right account you'll get a 404 (because they don't want to leak whether you activated that feature or not)